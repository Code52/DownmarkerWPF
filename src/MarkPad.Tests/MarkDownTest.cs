using ApprovalTests;
using ApprovalTests.Reporters;
using MarkPad.Document;
using Xunit;

namespace MarkPad.Tests
{
	//[UseReporter(typeof(FileLauncherReporter))]
	public class MarkDownTest
	{
		[Fact]
		public void TestParsing()
		{
			var text = @"# This is a header block

With a Url  [HyperLink](http://code52.org/DownmarkerWPF/) 
and some **bold** formatting and *italics*

## And a subheader

 - With a bullet point 1
 - And bullet point 2
 
 Finally there is a
 
    code { block;} =  here
    
Of course we could do `code` in the middle of the page";
			var output = DocumentParser.Parse(text);
			Approvals.VerifyHtml(output);
		}
	}
}