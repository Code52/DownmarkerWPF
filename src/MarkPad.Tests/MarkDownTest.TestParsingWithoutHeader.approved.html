<html>
<head>

</head>
<body>
<p>This Doesn't have a header</p>


<script type='text/javascript'>
	window.onload = function(){
		var links = document.getElementsByTagName('a');
		for (var i = 0; i < links.length; i++) {
			var l = links[i];
			if (l.getAttribute('href')) l.target = '_blank';
		}
	};
</script>
			
</body>
</html>